pydicom/tests/test_charset.py

```diff
@@ -435,11 +436,26 @@ def test_japanese_multi_byte_personname(self):
             ds_out = dcmread(fp)
             assert original_string == ds_out.PatientName.original_string
 
+        japanese_pn = PersonName3(u"Mori^Ogai=森^鷗外=もり^おうがい")
```

Comment:
Finally we are seeing new names here, not only Yamada Tarou from the standard example ;)

Reply:
I'll give @darcymason a chance to have another look, otherwise I will probably merge tomorrow.